AJS.Web/Areas/Identity/Pages/Account/Register.cshtml.cs

```diff
@@ -46,13 +46,13 @@ public class RegisterModel : PageModel
 
         public class InputModel
         {
-            [Required]
-            [EmailAddress]
+            [Required(ErrorMessage = "Email is required")]
+            [EmailAddress(ErrorMessage = "Email is not valid")]
             [Display(Name = "Email")]
             public string Email { get; set; }
 
-            [Required]
-            [StringLength(100, ErrorMessage = "The {0} must be at least {2} and at max {1} characters long.", MinimumLength = 6)]
+            [Required(ErrorMessage = "Password is required")]
+            [StringLength(100, ErrorMessage = "RegisterPasswordErrorMessage", MinimumLength = 6)]
```

Comment:
"RegisterPasswordErrorMessage", must be changed
`  [StringLength(100, ErrorMessage = "RegisterPasswordErrorMessage", MinimumLength = 6)]`